### PR TITLE
I34 socket notifications

### PR DESCRIPTION
**Pull request by spencersolomon6**
Added tests for follows service. Added support on the frontend for receiving socket messages and refreshing the notification state.

### CODE REVIEW

File: src/tests/users-service.test.js [hunk 1]
```diff
@@ -2,8 +2,7 @@ import {
   createUser,
```
Comment (sabrasisler) [inline src/tests/users-service.test.js:2]: Why are the tests from users-service being deleted?

**Review by sabrasisler — APPROVED** [review]
Looks good, I'm just wondering why you deleted the tests from User-service

File: src/tests/users-service.test.js (hunk 1 above)
Comment (spencersolomon6) [reply src/tests/users-service.test.js:2]: They were just moved into follows-service.test.js. The follow method was originally a part of the users-service, but it was refactored to have its own service and thus test file.

**Review by tmudunuri — APPROVED** [review]
Looks good.

**Review by creativediego — APPROVED** [review]

**Pull request merged**